packages/beacon-node/src/db/repositories/backfillState.ts

```diff
@@ -0,0 +1,29 @@
+import {ContainerType, ListUintNum64Type, ValueOf} from "@chainsafe/ssz";
+import {ChainForkConfig} from "@lodestar/config/lib";
```

Comment:
When importing its important to require the TS source not the built JS code
```suggestion
import {ChainForkConfig} from "@lodestar/config";
```